components/web/php/php-7_3/Makefile

```diff
@@ -0,0 +1,236 @@
+#CONFIGURE_OPTIONS     +=	--with-png-dir=$(ECPREFIX)
+#CONFIGURE_OPTIONS     +=	--with-zlib-dir=$(ECPREFIX)
+#CONFIGURE_OPTIONS     +=	--with-xpm-dir=$(ECPREFIX)
+#CONFIGURE_OPTIONS     +=	--with-libexpat-dir=$(ECPREFIX)
+#CONFIGURE_OPTIONS     +=	--with-openssl-dir=$(ECPREFIX)
+
+# Disabled extensions & Options
+CONFIGURE_OPTIONS   +=	--disable-static
+CONFIGURE_OPTIONS   +=	--disable-dba
+CONFIGURE_OPTIONS   +=	--disable-debug
+CONFIGURE_OPTIONS   +=	--disable-libgcc
+CONFIGURE_OPTIONS   +=	--without-dbm
+
+# Enabled extensions
+CONFIGURE_OPTIONS   +=	--enable-cli
+CONFIGURE_OPTIONS   +=	--enable-cgi
+CONFIGURE_OPTIONS   +=	--enable-shared
+CONFIGURE_OPTIONS   +=	--enable-filter
+CONFIGURE_OPTIONS   +=	--enable-gd-jis-conv
+CONFIGURE_OPTIONS   +=	--enable-short-tags
+CONFIGURE_OPTIONS   +=	--enable-pcntl
+CONFIGURE_OPTIONS   +=	--enable-shmop
+CONFIGURE_OPTIONS   +=	--enable-sysvmsg
+CONFIGURE_OPTIONS   +=	--enable-sysvsem
+CONFIGURE_OPTIONS   +=	--enable-sysvshm
+CONFIGURE_OPTIONS   +=	--enable-simplexml
+CONFIGURE_OPTIONS   +=	--enable-hash
+CONFIGURE_OPTIONS   +=	--enable-session
+CONFIGURE_OPTIONS   +=	--enable-opcache
+
+# Shared Extensions
+CONFIGURE_OPTIONS   +=	--enable-bcmath=shared
+CONFIGURE_OPTIONS   +=	--enable-json=shared
+CONFIGURE_OPTIONS   +=	--enable-libxml=shared
+CONFIGURE_OPTIONS   +=	--enable-ctype=shared
+CONFIGURE_OPTIONS   +=	--enable-calendar=shared
+CONFIGURE_OPTIONS   +=	--enable-tokenizer=shared
+CONFIGURE_OPTIONS   +=	--enable-exif=shared
+CONFIGURE_OPTIONS   +=	--enable-dom=shared
+CONFIGURE_OPTIONS   +=	--enable-ftp=shared
+CONFIGURE_OPTIONS   +=	--enable-intl=shared
+CONFIGURE_OPTIONS   +=	--enable-pdo=shared
+CONFIGURE_OPTIONS   +=	--enable-mbstring=shared
+CONFIGURE_OPTIONS   +=	--enable-xmlreader=shared
+CONFIGURE_OPTIONS   +=	--enable-xmlwriter=shared
+CONFIGURE_OPTIONS   +=	--enable-soap=shared
+CONFIGURE_OPTIONS   +=	--enable-sockets=shared
+CONFIGURE_OPTIONS   +=	--enable-wddx=shared
+CONFIGURE_OPTIONS   +=	--enable-zip=shared
+
+CONFIGURE_OPTIONS += $(CONFIGURE_OPTIONS.$(BITS))
+
+# Installation Arguments
+COMPONENT_INSTALL_ARGS  += INSTALL_ROOT=$(PROTO_DIR)
+
+# Pre-Install Actions
+COMPONENT_PRE_INSTALL_ACTION+= \
+	  ( $(MKDIR) -p $(PROTO_DIR)/$(AP_SYSCONFDIR)/conf.d ; \
+	  echo "Creating fake modules.load and httpd.conf" ; \
+	  echo "\# LoadModule foo_module libexec/mod_foo.so" >> $(PROTO_DIR)/$(AP_SYSCONFDIR)/conf.d/modules.load ; \
+	  echo "\# " >> $(PROTO_DIR)/$(AP_SYSCONFDIR)/conf.d/modules.load ; \
+	  echo "LoadModule foo_module libexec/mod_foo.so " >> $(PROTO_DIR)/$(AP_SYSCONFDIR)/conf.d/modules.load ; \
+          cp $(PROTO_DIR)/$(AP_SYSCONFDIR)/conf.d/modules.load  $(PROTO_DIR)/$(AP_SYSCONFDIR)/httpd.conf \
+)
+
+# Environment Variables
+COMPONENT_BUILD_ENV+=	CC="$(CC) $(CC_BITS)" CXX="$(CXX)" CFLAGS="$(CFLAGS)" LDFLAGS="$(LDFLAGS)"
+CONFIGURE_ENV +=     EXTENSION_DIR=$(PHP_PREFIX)/extensions
+CONFIGURE_ENV.64 +=     ICU_CONFIG=/usr/bin/amd64/icu-config
```

Comment:
Should we switch that to CONFIGURE_ENV only?

Reply:
well, it's too amd64-specific, don't know.

Reply:
True. Ship it